manifests/u/Ubiquiti/UniFiNetworkServer/8.2.93/Ubiquiti.UniFiNetworkServer.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Ubiquiti.UniFiNetworkServer
+PackageVersion: 8.2.93
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-06-03
+AppsAndFeaturesEntries:
+  - DisplayName: Ubiquiti UniFi
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
Installers:
```

Reply:
Seems it's the actual reason. Verified in my `amd64` _Windows Sandbox_.

![image](https://github.com/microsoft/winget-pkgs/assets/49941141/91ad051a-2f7d-4ceb-8cfc-8ef8278cbe60)
![image](https://github.com/microsoft/winget-pkgs/assets/49941141/560a3a1d-0194-48e2-a8ce-99a736f63846)